enarx-keep/tests/integration_tests.rs

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::io::Read;
+use std::path::Path;
+use std::process::{Command, Stdio};
+use std::time::Duration;
+use wait_timeout::ChildExt;
+
+const CRATE: &str = env!("CARGO_MANIFEST_DIR");
+const KEEP_BIN: &str = "/target/debug/enarx-keep";
+const OUT_DIR: &str = env!("OUT_DIR");
+const TEST_BINS_OUT: &str = "bin";
+const TIMEOUT_SECS: u64 = 5;
+
+/// Returns a handle to a child process through which output (stdout, stderr) can
+/// be accessed.
+fn run_test(bin: &str, status: Option<i32>) -> std::process::Child {
```

Comment:
Since we always specify an exit status...

```suggestion
fn run_test(bin: &str, status: i32) -> std::process::Child {
```